tests/test_indexer.py

```diff
@@ -115,14 +116,19 @@ def tearDown(self):
         self.storageHelper = None
 
     def test_process_new_indexable_object(self):
-        bundle_key = self.load_test_data_bundle_for_path("fixtures/smartseq2/paired_ends")
-        sample_event = self.create_sample_bundle_created_event(bundle_key)
-        self.process_new_indexable_object(sample_event, logger)
+        bundle_key = self.insert_and_process_new_indexable_object()
         search_results = self.get_search_results(smartseq2_paired_ends_query, 1)
         self.assertEqual(1, len(search_results))
         self.verify_index_document_structure_and_content(search_results[0], bundle_key,
                                                          files=smartseq2_paried_ends_indexed_file_list)
 
+    def test_process_removed_object(self):
```

Comment:
Please add two additional tests:

1.  The key is not a valid bundle key
2. The key is a valid bundle key, but the bundle is not indexed.